src/status_im2/contexts/chat/messages/list/view.cljs

```diff
@@ -121,8 +120,9 @@
                          chat-id show-input? message-pin-enabled edit-enabled in-pinned-view? can-delete-message-for-everyone?]}]
   [rn/view {:style (when (and platform/android? (not in-pinned-view?)) {:scaleY -1})}
    (if (= type :datemark)
-     ;; TODO (flexsurfer) implement and use date divider component https://github.com/status-im/status-mobile/issues/14523
-     [message-datemark/chat-datemark (:value message)]
+     [rn/touchable-without-feedback
+      {:on-press rn/dismiss-keyboard!}
```

Comment:
hmm, I'm still not convinced this needs to be here. 
if you click on the chat screen the keyboard should disappear. Did you discuss this with someone from UX or it's just an assumption you made that it needs this?

@OmarBasem perhaps you know when/why the keyboard should be dismissed in messages.

Reply:
The old component had that logic. See https://github.com/status-im/status-mobile/blob/develop/src/status_im/ui/screens/chat/message/datemark.cljs. That's why I implemented it. And @flexsurfer [suggested](https://github.com/status-im/status-mobile/pull/14546#discussion_r1049244965) moving it outside of the divider-date component.

Reply:
Hi @ajayesivan, I'm aware of why you moved it. My question is broader than that.

Why does this component need to know about dismissing the keyboard at all?

Surely clicking anywhere on the chat window should dismiss the keyboard, so why does this behaviour need to be explicitly set on the divider-date?

Did some developer or one of the designers say that the divider-date needs this functionality? Or is this just an assumption that it needs this?

Reply:
Thanks @ajayesivan, it seems in the component you sent that this on-press was added to fix some issues with scrolling:
https://github.com/status-im/status-mobile/issues/7799

If possible can you see if this issue returns if you try removing this code. if there is no difference without the code then we can remove it. otherwise keep it 👍

Reply:
thanks, looks good to merge once you've checked 👍

Reply:
@J-Son89 Works perfectly without the on-press and hence removed it.

Reply:
Nice one! We can merge this so :)